sirepo/auth_role_moderation.py

```diff
@@ -142,8 +141,32 @@ def _send_request_email(info):
         return self.reply_ok()
 
 
+def control_for_user(uid, role):
```

Comment:
possibly `check_user()`?

Reply:
4980a3bd5 I think it is fine. I prefixed it with raise_ since the other calls in that method are also prefixed with raise_